### ISSUE TITLE
No (obvious) way to insert literal text that conflicts with an AutoKey "special key"

### ISSUE THREAD

**Issue by ernstki** [autokey/autokey#110]
## Classification:

UI/Usability

## Reproducibility:

Always

## Summary

There's no obvious way to override the special handling of literal text in a `keyString`, so something as benign as trying to wrap clipboard text in `<code></code>` tags or using `keyboard.send_keys` with arbitrary clipboard text possibly containing tokens that AutoKey considers "special" will not work as expected.

A secondary problem is that while the ability to parse raw keycodes within keyStrings is probably useful, it doesn't seem to be [terribly well-documented][api] (as in, you'd have to inspect [the source code][mediator] to discover it).

## Steps to Reproduce

Create a script with the following contents and run it

```python
keyboard.send_key("<code>test</code>")
```

## Expected Results

Because keycode handling in `keyString`s appears to be undocumented, one would expect that the literal text `<code>test</code>` would show up in the active document / text area.

## Actual Results

Nothing. You would never know anything was wrong (or right) without running `autokey-gtk -l` and seeing the following error message, due to the improper parsing of the keyString starting with `<code` (from `lib/iomediator_Key.py` [around line 67](../blob/master/lib/autokey/iomediator_Key.py#L67)):

```
2017-11-27 21:01:03,548 DEBUG - interface - Send special key: ['<code>test</code>']
2017-11-27 21:01:03,548 ERROR - interface - Error in X event loop thread
Traceback (most recent call last):
  File "/home/user/.local/lib/python3.5/site-packages/autokey/interface.py", line 146, in __eventLoop
    method(*args)
  File "/home/user/.local/lib/python3.5/site-packages/autokey/interface.py", line 714, in __sendKey
    self.__sendKeyCode(self.__lookupKeyCode(keyName))
  File "/home/user/.local/lib/python3.5/site-packages/autokey/interface.py", line 961, in __lookupKeyCode
    return int(char[5:-1])
ValueError: invalid literal for int() with base 10: '>test</code'
```

## Version

AutoKey-Py3 version 0.93.10, installed via pip.

Distro: elementaryOS Loki (Ubuntu 16.04 Xenial)

## Notes

I respectfully suggest adding a `keyboard.send_keys_raw` or `keyboard.send_keys_unparsed` method, which does not parse the `keyString` for "special keys."

### Workaround

```python
# split up part of "special key" across two calls to 'keyboard.send_key'
keyboard.send_key("<")
keyboard.send_key("code>test</code>")
```

[api]: https://autokey.github.io/lib.scripting.Keyboard-class.html
[mediator]: ../blob/master/lib/autokey/iomediator_Key.py

**Comment by luziferius** [comment]
Fixed in https://github.com/autokey/autokey/commit/c8a2d43611dadcd6b6d75bb0999614e821e43050.
(The commit message only mentions phrases, but it also works in scripts when using `keyboard.send_keys`.)
This will be included in the next version.

You still can’t include a literal `<code40>` in texts, but everything that is not exactly `<code[any_number_of_digits]>`, like `<code>`, `</code>`, `<code name="whatever">` can be typed and included in phrases.

**Comment by luziferius** [comment]
A fix is released in 0.95.4.
@ernstki The fix released does not cover all corner cases, as stated in the last comment, but it covers your use case. You may want to update to 0.95.4.

(There are debian packages attached to the release, which should also work on Ubuntu 16.04 releases.)

**Comment by ernstki** [comment]
@luziferius Thanks for your efforts!
